conf/modules.config

```
@@ -89,7 +169,16 @@ process {
         ]
     }
 
+    withName: 'DROP.*FRASER_FITHYPERPARAMS' {
+        cpus = { 12 * task.attempt }
```

this one seems worth more cpus. Maybe 30?
```suggestion
        cpus = { 30 * task.attempt }
```